bind/c-ptexture.cpp

```diff
@@ -383,15 +383,23 @@ template <class T> struct PtexPtr {
     auto reset(T* ptr) -> void;
 } CPPMM_OPAQUEBYTES; // struct PtexPtr
 
-// TODO: fill in explicit instantiations, e.g.:
-// template class PtexPtr<int>;
-// using PtexPtrInt = Ptex::v2_2::PtexPtr<int>;
+// PtexPtr explicit instantiations
+//template class PtexPtr<Ptex::PtexCache>;
+using PtexCachePtr = PtexPtr<Ptex::PtexCache>;
 
-} // namespace v2_3
+//template class PtexPtr<Ptex::PtexMetaData>;
+using PtexMetaDataPtr = PtexPtr<Ptex::PtexMetaData>;
+
+//template class PtexPtr<Ptex::PtexTexture>;
+using PtexTexturePtr = PtexPtr<Ptex::PtexTexture>;
```

Comment:
Heads up -- as soon as I uncomment the `template class` declarations above I get typedef redefinition errors:

```
ptex-c/./c-ptexture.h:239:56: error: typedef redefinition with different types
      ('Ptex__v2_4__PtexPtr_Ptex__v2_4__PtexTexture__t' (aka
      'Ptex__v2_4__PtexPtr_Ptex__v2_4__PtexTexture__t_s') vs
      'Ptex__v2_4__PtexPtr_Ptex__v2_4__PtexCache__t' (aka
      'Ptex__v2_4__PtexPtr_Ptex__v2_4__PtexCache__t_s'))
typedef Ptex__v2_4__PtexPtr_Ptex__v2_4__PtexTexture__t Ptex_PtexPtr_t;
                                                       ^
ptex-c/./c-ptexture.h:234:54: note: previous definition is here
typedef Ptex__v2_4__PtexPtr_Ptex__v2_4__PtexCache__t Ptex_PtexPtr_t;
                                                     ^
```

I left them commented out for now to get these improvements in. I'm probably doing something wrong there ;-)